ChaosMod/Components/LuaScripts.cpp

```diff
@@ -599,6 +599,8 @@ LuaScripts::ParseScriptReturnReason LuaScripts::ParseScriptRaw(std::string scrip
 		E("GetAllPeds", GetAllPedsArray),
 		E("CreatePoolPed", CreatePoolPed),
 
+		E("TeleportPlayer", [](float x, float y, float z, bool noOffset) { TeleportPlayer(x, y, z, noOffset); }),
```

Comment:
```suggestion
		E("TeleportPlayer", TeleportPlayer),
```
No reason to wrap it with the previous changes